.env.docker

```diff
@@ -0,0 +1,5 @@
+# Sample docker-compose input
+ENV=production
```

Comment:
A bit misleading key - can be confused with '.env'
Would not ENVIRONMENT, or DEPLOYMENT be better?

Reply:
I followed your suggestion from the Architectura doc :-) But it's a good point. TBH I am not sure if it's even needed, but maybe for some debugging purposes.

I don't think that ENVIRONMENT is a good enough distinction. What if we went with CONTEXT? That's a naming convention that is used by Circle CI and Netlify.

Reply:
CONTEXT is quite generic IMO ( what is not a context :-) ? ), but I am OK with that to resolve the conflict with ENV.